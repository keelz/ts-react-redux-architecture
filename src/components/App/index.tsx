import * as React from 'react';

class App extends React.Component {
  public render() {
    return (
      <div>
        <header>
          <h1>Welcome to React</h1>
        </header>
        <p>
          To get started, edit <code>src/App.tsx</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
