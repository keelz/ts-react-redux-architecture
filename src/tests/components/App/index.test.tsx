// src/tests/components/App/index.tst.tsx
import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from '../../../components/App';

describe('App', () => {
  describe('snapshot', () => {
    it('renders correctly', () => {
      const tree = enzyme.shallow(<App />);
      expect(shallowToJson(tree)).toMatchSnapshot();
    });
  });
});
