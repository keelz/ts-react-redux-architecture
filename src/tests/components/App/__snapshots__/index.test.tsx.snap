// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
    />
    <h1
      className="App-title"
    >
      Welcome to React
    </h1>
  </header>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.tsx
    </code>
     and save to reload.
  </p>
</div>
`;
